dataset_generation/hi_wikisplit/Scraper.py

```diff
@@ -0,0 +1,174 @@
+import logging
+import requests
```

Comment:
can we also please run pylint on each of the python files.

Reply:
yes, i'll do that.